roles/openshift_node_dnsmasq/files/networkmanager/99-origin-dns.sh

```diff
@@ -112,7 +112,7 @@ EOF
       fi
       sed -e '/^nameserver.*$/d' /etc/resolv.conf >> ${NEW_RESOLV_CONF}
       echo "nameserver "${def_route_ip}"" >> ${NEW_RESOLV_CONF}
-      if ! grep -q 'search.*cluster.local' ${NEW_RESOLV_CONF}; then
+      if ! grep -q 'search.*/scluster.local' ${NEW_RESOLV_CONF}; then
```

Comment:
I think we should template this file now so we can use `{{ openshift.common.dns_domain }}`

Reply:
to move dispatcher script to the template and use ansible to manage this file script?

Reply:
Yeah, so that we can check for the proper dns zone if it's been reconfigured.

Reply:
lets merge this as hotfix and I can do this sometimes in next few weeks when I have time if this is not in rush. We can track it in the issues or trello :)